### DISCUSSION TITLE
Allow for projects=all when running the install generator in non-interactive mode

### DISCUSSION THREAD

**Discussion by jdgeier** [jscutlery/semver#491] [Ideas]
Hello, I know that when I run the install generator it basically sets everything to all by default but I have to run the generator in interactive mode in order for that to work correctly. I'd like to run the command prior to tagging an affected:version to insure all packages in the project get built, tagged, and deployed. Basically I would like to run the following:

`npx nx g @jscutlery/semver:install --interactive=false --enforce-conventional-commits --projects=all`

**Comment by edbzn** [comment]
Yeah, must be usefull. I don't think we need to introduce --interactive=false as passing --projects automatically turn off the interactive mode.
